dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -725,7 +741,7 @@ private boolean isValidCwl(String content, Yaml yaml) {
             final Object cwlVersion = mapping.get("cwlVersion");
 
             if (cwlVersion != null) {
-                final boolean equals = cwlVersion.toString().startsWith("v1");
+                final boolean equals = cwlVersion.toString().startsWith(CWLHandler.CWL_VERSION_PREFIX);
```

Comment:
equals != startsWith

Reply:
[Modified](https://github.com/dockstore/dockstore/blob/76d246baa2dabe6b004914606056c5645a7acb12/dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java#L746). Changed to startsWith (in both locations, see comment above).